tutorials/Working_with_fNIRS_data.py

```diff
@@ -0,0 +1,124 @@
+"""
+.. _tut_fnirs_data
+
+Use MNE with fNIRS data
+=======================
+# Author: Matteo Caffini  
+# Contact: <matteo.caffini@unitn.it>
+
+"""
+import numpy as np
+
+import mne
+
+raw_ndarray = np.load('/home/matteo/raw_ndarray.npy')
+
+###############################################################################
+# MNE Python was implemented with EEG and MEG in mind, but can be useful to
+# process fNIRS data as #well. A common fNIRS data set (similarly to EEG and
+# MEG data) contains data recorded over time(a.k.a. samples) at many locations
+# on the scalp (a.k.a. channels).
+# Usually in fNIRS one looks for brain responses to external sensorial
+# stimulations (visual, acoustic, etc.). In order to do so, block design
+# studies are usually employed and data are later epoched to get evoked
+# responses.
+
+###############################################################################
+# fNIRS raw data come in various formats and the best way to dive into MNE is
+# to deal with your own format first and fit your data into a 2D numpy array
+# [channels x samples]. Once you prepared your time series for HbO and HbR
+# concentrations, you can create a RawArray instance from the data array using
+# mne.io.RawArray. If you have a time series of events synchronous with your
+# data you can append this as a last row to your array.
+# 
+# In this tutorial, I don't preprocess the data. If your fNIRS device outputs
+# data in the form of photon fluences at detectors, you can compute HbO and HbR
+# concentrations using the modified Lambert-Beer model.
```

Comment:
Can you provide some link either to a paper (in References section) or e.g. Wikipedia?